authenticator/src/main/kotlin/com/bitwarden/authenticator/data/platform/manager/SdkClientManagerImpl.kt

```diff
@@ -1,12 +1,27 @@
 package com.bitwarden.authenticator.data.platform.manager
 
+import com.bitwarden.core.ClientManagedTokens
 import com.bitwarden.sdk.Client
 
+/**
+ * The token provider to pass to the SDK.
+ */
+class Token : ClientManagedTokens {
```

Comment:
Since the actual implementation will always return null for Authenticator, we can define this as a private inner class of `SdkClientManagerImpl` if we can't pass `null` for `tokenProvider`.